impl/StreamPutApiImpl.cpp

```diff
@@ -23,7 +23,13 @@ StreamPutApiImpl::StreamPutApiImpl(Pistache::Address addr)
     : StreamPutApi(addr)
     { }
 
-void StreamPutApiImpl::stream(const Pistache::Http::Header::Collection& headers, const std::string body, const Pistache::Optional<std::string> &path, const Pistache::Optional<std::string> &offset, const Pistache::Optional<std::string> &limit, Pistache::Http::ResponseWriter &response) {
+void StreamPutApiImpl::stream(const Pistache::Http::Header::Collection& headers,
+                              const std::string& body,
+                              const std::string& path,
+                              const Pistache::Optional<std::string>& offset,
```

Comment:
now that limit is not optional for streamGet... do we want to keep the same order for streamPut?  aka, limit before offset in the signature?   for consistency?

Reply:
yes. that change is already included in the current PR. your comment is just referencing the PUT endpoint.

Reply:
yes, wondering if having Put and Get have different parameter orderings will confuse someone.